{
  "entity": {
    "location": {
      "namespaceTitle": "Fuse",
      "namespaceUri": "fuse",
      "packageName": "Fuse.Common",
      "packageVersion": "1.8.0-rc2"
    },
    "parameters": [
      {
        "name": "msg",
        "href": "uno/string",
        "title": "string",
        "fullyQualifiedTitle": "string"
      },
      {
        "name": "ex",
        "href": "uno/exception",
        "title": "Exception",
        "fullyQualifiedTitle": "Uno.Exception"
      },
      {
        "name": "obj",
        "href": "uno/object",
        "title": "object",
        "fullyQualifiedTitle": "object"
      },
      {
        "name": "filePath",
        "href": "uno/string",
        "title": "string",
        "fullyQualifiedTitle": "string"
      },
      {
        "name": "lineNumber",
        "href": "uno/int",
        "title": "int",
        "fullyQualifiedTitle": "int"
      },
      {
        "name": "memberName",
        "href": "uno/string",
        "title": "string",
        "fullyQualifiedTitle": "string"
      }
    ],
    "id": {
      "id": "fuse/diagnostics/unknownexception_7b89b1e6",
      "parentId": "fuse/diagnostics",
      "type": "Method",
      "modifiers": [
        "public",
        "static"
      ]
    },
    "uri": {
      "idUri": "fuse/diagnostics/unknownexception_7b89b1e6",
      "href": "fuse/diagnostics/unknownexception_7b89b1e6"
    },
    "titles": {
      "pageTitle": "Diagnostics.UnknownException Method (string, Exception, object, string, int, string)",
      "navigationTitle": "UnknownException Method (string, Exception, object, string, int, string)",
      "fullTitle": "Fuse.Diagnostics.UnknownException",
      "indexTitle": "UnknownException(string, Exception, object, string, int, string)",
      "fullyQualifiedIndexTitle": "Fuse.Diagnostics.UnknownException(string, Uno.Exception, object, string, int, string)"
    },
    "syntax": {
      "uno": "public static voidUnknownException(string msg, Uno.Exception ex, object obj, string filePath = \"\"\"\", int lineNumber = 0, string memberName = \"\"\"\") {}"
    },
    "comment": {
      "brief": "Used when an expected exception is caught and otherwise ignored (processing continues).\nInternal implies the immediate cause is not known and it cannot be attributed to a user error.",
      "full": "Used when an expected exception is caught and otherwise ignored (processing continues).\nInternal implies the immediate cause is not known and it cannot be attributed to a user error."
    }
  },
  "tableOfContents": {}
}